Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
```

Comment:
I take it you're not a fan of including the `internal` keyword, even if that's (currently) the default? That's cool, just wondering.

Tangentially, do you think it'd be worth it to use a single OSS Swift style guide across the repos in the Quick org? I could be convinced either way.